src/config/main.js

```diff
@@ -2,68 +2,77 @@
 // This is the main configuration file where universal and default settings should be placed.
 // These settins can then be imported anywhere in the app as they are exported at the botom of the file.
 
-import { jsPsych } from 'jspsych-react'
-import _ from 'lodash'
-import { eventCodes } from './trigger'
-
+import { jsPsych } from "jspsych-react";
+import _ from "lodash";
+import { eventCodes } from "./trigger";
+import { verifyProlific } from "../lib/utils";
 
 require("dotenv").config();
 
 // mapping of letters to key codes
 const keys = {
-	"A": 65,
-	"B": 66,
-	"C": 67,
-	"F": 70,
-	"J": 74,
-	"space": 32
-}
+  A: 65,
+  B: 66,
+  C: 67,
+  F: 70,
+  J: 74,
+  space: 32,
+};
 
 // audio codes
 const audioCodes = {
-	frequency: 100*(eventCodes.open_task - 9),
-	type: 'sine'
-}
-
-// is this mechanical turk?
-const MTURK = (!jsPsych.turk.turkInfo().outsideTurk)
-const AT_HOME = (process.env.REACT_APP_AT_HOME === 'true')
-const VIDEO =  (process.env.REACT_APP_VIDEO === 'true')
-var IS_ELECTRON = (process.env.REACT_APP_IS_ELECTRON === 'true')
-const PROLIFIC = (process.env.REACT_APP_PROLIFIC === 'true')
+  frequency: 100 * (eventCodes.open_task - 9),
+  type: "sine",
+};
 
-// is this firebase?
-const FIREBASE = (process.env.REACT_APP_FIREBASE === 'true')
+const MTURK = !jsPsych.turk.turkInfo().outsideTurk;
+let PROLIFIC = verifyProlific() && !MTURK;
+let IS_ELECTRON = true;
+let FIREBASE = process.env.REACT_APP_FIREBASE === "true";
 
 // Could work on this logic
+
 try {
-	window.require("electron");
-  } catch {
-	window.alert("Not electron")
-	IS_ELECTRON = false
-  }
+  window.require("electron");
+} catch {
+  IS_ELECTRON = false;
+}
 
+// these variables depend on IS_ELECTRON
+// whether or not to ask the participant to adjust the volume
+const VOLUME = process.env.REACT_APP_VOLUME === "true" && IS_ELECTRON;
```

Comment:
does the volume one really need to be electron specific?

Reply:
from what I know, volume is clinic specific and clinic is always electron. Do I have that right?

Reply:
while that's currently true, I don't think there's a technical reason volume has to be electron.  whereas with the photodiode and event marker those require the electron integration